client/state/plugins/installed/selectors.js

```diff
@@ -89,7 +89,7 @@ export function getPlugins( state, siteIds, pluginFilter ) {
 		pluginList = filter( pluginList, _filters[ pluginFilter ] );
 	}
 
-	return values( sortBy( pluginList, ( item ) => item.slug.toLowerCase() ) );
```

Comment:
Did you mean to remove the call here? It looks like `pluginList` is an object based on the `reduce` call above 🤔 I think we should keep the `Object.values` call here right? Otherwise the API of this method is changing to return an object instead of an array.

Reply:
`sortBy` returns an array though. So why do we need to convert it to an array again?

Reply:
Oh, duh! Of course not 🙂